[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:329 



[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:330 Model  Controller           Cloud/Region         Version  SLA          Timestamp
test   localhost-localhost  localhost/localhost  3.5.3    unsupported  03:19:41Z

App                       Version  Status   Scale  Charm                     Channel        Rev  Exposed  Message
failover                           active       2  opensearch                                 2  no       
grafana-agent                      blocked      4  grafana-agent             latest/edge    286  no       Missing ['grafana-cloud-config']|['grafana-dashboards-provider']|['logging-consumer']|['send-remote-write'] for cos-a...
main                               active       1  opensearch                                 1  no       
opensearch                         active       1  opensearch                                 0  no       
self-signed-certificates           active       1  self-signed-certificates  latest/stable  155  no       

Unit                         Workload     Agent      Machine  Public address  Ports     Message
failover/0                   active       idle       2        10.34.139.178   9200/tcp  
  grafana-agent/1            maintenance  executing           10.34.139.178             (install) Installing grafana-agent snap
failover/1*                  active       executing  3        10.34.139.5     9200/tcp  
  grafana-agent/0*           blocked      idle                10.34.139.5               Missing ['grafana-cloud-config']|['grafana-dashboards-provider']|['logging-consumer']|['send-remote-write'] for cos-a...
main/1*                      active       executing  4        10.34.139.26    9200/tcp  (set-password) 
  grafana-agent/2            blocked      executing           10.34.139.26              Missing ['grafana-cloud-config']|['grafana-dashboards-provider']|['logging-consumer']|['send-remote-write'] for cos-a...
opensearch/0*                active       executing  1        10.34.139.183   9200/tcp  
  grafana-agent/3            blocked      idle                10.34.139.183             Missing ['grafana-cloud-config']|['grafana-dashboards-provider']|['logging-consumer']|['send-remote-write'] for cos-a...
self-signed-certificates/0*  active       idle       0        10.34.139.180             

Machine  State    Address        Inst id        Base          AZ  Message
0        started  10.34.139.180  juju-02639c-0  ubuntu@22.04      Running
1        started  10.34.139.183  juju-02639c-1  ubuntu@22.04      Running
2        started  10.34.139.178  juju-02639c-2  ubuntu@22.04      Running
3        started  10.34.139.5    juju-02639c-3  ubuntu@22.04      Running
4        started  10.34.139.26   juju-02639c-4  ubuntu@22.04      Running

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


03:19:41 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: main - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: main active -- message: None
		main-1.625* -- (10.34.139.26) -- [executing (since: 03:19:36)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


03:19:53 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: main - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: main active -- message: None
		main-1.625* -- (10.34.139.26) -- [idle (since: 03:19:46)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


03:20:05 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: main - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: main active -- message: None
		main-1.625* -- (10.34.139.26) -- [idle (since: 03:19:46)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


03:20:16 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: main - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: main active -- message: None
		main-1.625* -- (10.34.139.26) -- [idle (since: 03:19:46)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


03:20:28 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: main - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: main active -- message: None
		main-1.625* -- (10.34.139.26) -- [idle (since: 03:19:46)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


03:20:39 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: main - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: main active -- message: None
		main-1.625* -- (10.34.139.26) -- [idle (since: 03:19:46)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


03:20:50 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: main - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: main active -- message: None
		main-1.625* -- (10.34.139.26) -- [idle (since: 03:19:46)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


03:21:02 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.4c2* -- (10.34.139.183) -- [idle (since: 03:19:57)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


03:21:16 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:348 03:21:20 -- Waiting for model: complete.
[32mINFO    [0m pytest_operator.plugin:plugin.py:862 Model status:

Model  Controller           Cloud/Region         Version  SLA          Timestamp
test   localhost-localhost  localhost/localhost  3.5.3    unsupported  03:21:23Z

App                       Version  Status   Scale  Charm                     Channel        Rev  Exposed  Message
failover                           active       2  opensearch                                 2  no       
grafana-agent                      blocked      4  grafana-agent             latest/edge    286  no       Missing ['grafana-cloud-config']|['grafana-dashboards-provider']|['logging-consumer']|['send-remote-write'] for cos-a...
main                               active       1  opensearch                                 1  no       
opensearch                         active       1  opensearch                                 0  no       
self-signed-certificates           active       1  self-signed-certificates  latest/stable  155  no       

Unit                         Workload     Agent      Machine  Public address  Ports     Message
failover/0                   active       idle       2        10.34.139.178   9200/tcp  
  grafana-agent/1            maintenance  executing           10.34.139.178             (install) Installing grafana-agent snap
failover/1*                  active       idle       3        10.34.139.5     9200/tcp  
  grafana-agent/0*           blocked      idle                10.34.139.5               Missing ['grafana-cloud-config']|['grafana-dashboards-provider']|['logging-consumer']|['send-remote-write'] for cos-a...
main/1*                      active       idle       4        10.34.139.26    9200/tcp  
  grafana-agent/2            blocked      idle                10.34.139.26              Missing ['grafana-cloud-config']|['grafana-dashboards-provider']|['logging-consumer']|['send-remote-write'] for cos-a...
opensearch/0*                active       idle       1        10.34.139.183   9200/tcp  
  grafana-agent/3            blocked      idle                10.34.139.183             Missing ['grafana-cloud-config']|['grafana-dashboards-provider']|['logging-consumer']|['send-remote-write'] for cos-a...
self-signed-certificates/0*  active       idle       0        10.34.139.180             

Machine  State    Address        Inst id        Base          AZ  Message
0        started  10.34.139.180  juju-02639c-0  ubuntu@22.04      Running
1        started  10.34.139.183  juju-02639c-1  ubuntu@22.04      Running
2        started  10.34.139.178  juju-02639c-2  ubuntu@22.04      Running
3        started  10.34.139.5    juju-02639c-3  ubuntu@22.04      Running
4        started  10.34.139.26   juju-02639c-4  ubuntu@22.04      Running

[32mINFO    [0m pytest_operator.plugin:plugin.py:868 Juju error logs:

machine-0: 03:07:34 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-0: 03:07:34 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-self-signed-certificates-0: 03:07:34 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-3: 03:08:02 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-3: 03:08:02 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-failover-1: 03:08:02 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-1: 03:08:50 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-1: 03:08:50 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-opensearch-0: 03:08:50 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-2: 03:09:27 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-2: 03:09:27 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-failover-0: 03:09:28 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-4: 03:10:37 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-4: 03:10:37 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-main-1: 03:10:37 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
unit-main-1: 03:12:56 ERROR unit.main/1.juju-log certificates:15: err:  / out: keytool error: java.lang.Exception: Keystore file does not exist: /var/snap/opensearch/current/etc/opensearch/certificates/ca.p12

unit-main-1: 03:13:02 ERROR unit.main/1.juju-log Cannot connect to the OpenSearch server...
unit-main-1: 03:13:03 ERROR unit.main/1.juju-log Cannot connect to the OpenSearch server...
unit-main-1: 03:13:03 ERROR unit.main/1.juju-log Cannot connect to the OpenSearch server...
unit-main-1: 03:13:06 ERROR unit.main/1.juju-log Cannot connect to the OpenSearch server...
unit-main-1: 03:13:09 ERROR unit.main/1.juju-log Cannot connect to the OpenSearch server...
unit-main-1: 03:13:12 ERROR unit.main/1.juju-log Cannot connect to the OpenSearch server...
unit-main-1: 03:13:15 ERROR unit.main/1.juju-log Cannot connect to the OpenSearch server...
unit-failover-0: 03:14:07 ERROR unit.failover/0.juju-log certificates:16: err:  / out: keytool error: java.lang.Exception: Keystore file does not exist: /var/snap/opensearch/current/etc/opensearch/certificates/ca.p12

unit-failover-1: 03:14:07 ERROR unit.failover/1.juju-log certificates:16: err:  / out: keytool error: java.lang.Exception: Keystore file does not exist: /var/snap/opensearch/current/etc/opensearch/certificates/ca.p12

unit-opensearch-0: 03:14:11 ERROR unit.opensearch/0.juju-log certificates:17: err:  / out: keytool error: java.lang.Exception: Keystore file does not exist: /var/snap/opensearch/current/etc/opensearch/certificates/ca.p12

unit-failover-0: 03:14:13 ERROR unit.failover/0.juju-log Cannot connect to the OpenSearch server...
unit-failover-1: 03:14:13 ERROR unit.failover/1.juju-log Cannot connect to the OpenSearch server...
unit-failover-0: 03:14:14 ERROR unit.failover/0.juju-log Cannot connect to the OpenSearch server...
unit-failover-1: 03:14:15 ERROR unit.failover/1.juju-log Cannot connect to the OpenSearch server...
unit-failover-1: 03:14:17 ERROR unit.failover/1.juju-log opensearch-peers:3: Cannot connect to the OpenSearch server...
unit-opensearch-0: 03:14:17 ERROR unit.opensearch/0.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 03:14:18 ERROR unit.opensearch/0.juju-log Cannot connect to the OpenSearch server...
unit-failover-1: 03:14:19 ERROR unit.failover/1.juju-log opensearch-peers:3: Cannot connect to the OpenSearch server...
unit-opensearch-0: 03:14:20 ERROR unit.opensearch/0.juju-log opensearch-peers:0: Cannot connect to the OpenSearch server...
unit-opensearch-0: 03:14:21 ERROR unit.opensearch/0.juju-log peer-cluster:14: Cannot connect to the OpenSearch server...
unit-opensearch-0: 03:14:24 ERROR unit.opensearch/0.juju-log opensearch-peers:0: Cannot connect to the OpenSearch server...
unit-opensearch-0: 03:14:26 ERROR unit.opensearch/0.juju-log opensearch-peers:0: Cannot connect to the OpenSearch server...
unit-failover-1: 03:14:37 ERROR unit.failover/1.juju-log opensearch-peers:3: Cannot connect to the OpenSearch server...
unit-failover-1: 03:14:39 ERROR unit.failover/1.juju-log opensearch-peers:3: Cannot connect to the OpenSearch server...
unit-opensearch-0: 03:14:52 ERROR unit.opensearch/0.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 03:15:03 ERROR unit.opensearch/0.juju-log peer-cluster:14: Cannot connect to the OpenSearch server...
unit-opensearch-0: 03:15:05 ERROR unit.opensearch/0.juju-log peer-cluster:14: Cannot connect to the OpenSearch server...
unit-grafana-agent-1: 03:17:08 ERROR juju.worker.dependency "uniter" manifold worker returned unexpected error: preparing operation "install ch:amd64/jammy/grafana-agent-286" for grafana-agent/1: failed to download charm "ch:amd64/jammy/grafana-agent-286" from API server: download request with archiveSha256 length 0 not valid
unit-grafana-agent-2: 03:17:08 ERROR juju.worker.dependency "uniter" manifold worker returned unexpected error: preparing operation "install ch:amd64/jammy/grafana-agent-286" for grafana-agent/2: failed to download charm "ch:amd64/jammy/grafana-agent-286" from API server: download request with archiveSha256 length 0 not valid
unit-grafana-agent-0: 03:17:08 ERROR juju.worker.dependency "uniter" manifold worker returned unexpected error: preparing operation "install ch:amd64/jammy/grafana-agent-286" for grafana-agent/0: failed to download charm "ch:amd64/jammy/grafana-agent-286" from API server: download request with archiveSha256 length 0 not valid
unit-grafana-agent-3: 03:17:08 ERROR juju.worker.dependency "uniter" manifold worker returned unexpected error: preparing operation "install ch:amd64/jammy/grafana-agent-286" for grafana-agent/3: failed to download charm "ch:amd64/jammy/grafana-agent-286" from API server: download request with archiveSha256 length 0 not valid
unit-grafana-agent-1: 03:17:09 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
unit-grafana-agent-0: 03:17:09 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
unit-grafana-agent-2: 03:17:09 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
unit-grafana-agent-3: 03:17:09 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk

[32mINFO    [0m pytest_operator.plugin:plugin.py:947 Forgetting model main...